src/services/collection.service.ts

```
@@ -65,4 +65,50 @@ export class CollectionService {
       status: 200,
     };
   }
+
+
+  async getCompetencyCollections(level_competency = [], language = "en", contentType): Promise<any[]> {
+    let collectionIds = await this.collectionModel.aggregate([
```

ESLint

[error] 'collectionIds' is never reassigned. Use 'const' instead.

(prefer-const)